interfaces/matlab_experimental/OneDim/Sim1D.m

```diff
@@ -58,14 +58,9 @@ function delete(s)
 
         %% Sim1D Utility Methods
 
-        function display(s, fname)
+        function display(s)
```

Comment:
For the sake of consistency with other APIs, the name of the method should be `show`, see https://cantera.org/stable/python/onedim.html#cantera.Sim1D.show.

Reply:
`display()` is a built-in Matlab command, I think

Reply:
> `display()` is a built-in Matlab command, I think... So either this should be renamed as suggested, or this provides the implementation for the built-in capabilities. Either way, it should be addressed 😀

Other classes such as `ThermoPhase`, `Mixture`, and `Func1` use their own `display` method to override MATLAB's default. Should I change these as well?